segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/hdfs/HDFSStorageConfig.java

```diff
@@ -23,6 +23,7 @@
 public class HDFSStorageConfig {
     //region Config Names
 
+    public static final Property<Boolean> ENABLE_HDFS = Property.named("enableHdfs", Boolean.TRUE);
```

Comment:
This  Pull Request should not touch HDFSStorage, HDFSStorageConfig, or anything HDFS-related.